messagebus/messagebus.go

```diff
@@ -146,6 +153,44 @@ func (mb *MessageBus) MustRegister(p insolar.MessageType, handler insolar.Messag
 	}
 }
 
+func (mb *MessageBus) createWatermillMessage(ctx context.Context, parcel insolar.Parcel, ops *insolar.MessageSendOptions, currentPulse insolar.Pulse) *watermillMsg.Message {
```

Comment:
currentPulse - do we really need to pass this param? can we get it inside function from pulsestorage?